environment.mjs

```diff
@@ -0,0 +1,164 @@
+const SUPPORTED_ENVIRONMENTS = {
+  mainnet: {
+    name: 'Mainnet',
+    chainId: 1,
+    clientApiUrl: '',
+    optimistApiUrl: '',
+    optimistWsUrl: '',
+    web3WsUrl: '',
+  },
+  ropsten: {
+    name: 'Ropsten',
+    chainId: 3,
+    clientApiUrl: 'https://client1.testnet.nightfall3.com',
+    optimistApiUrl: 'https://optimist1.testnet.nightfall3.com',
+    optimistWsUrl: 'wss://optimist1-ws.testnet.nightfall3.com',
+    web3WsUrl: 'wss://ropsten1-ws.testnet.nightfall3.com',
```

Comment:
This is one of the things we need to think about. web3 cannot be passed like this because its private.